Formula/minidlna.rb

```diff
@@ -102,12 +102,14 @@ def plist
       log_dir=#{testpath}/.config/minidlna
     EOS
 
-    system sbin/"minidlnad", "-f", "minidlna.conf", "-p", "8081", "-P",
+    port = free_port
+
+    system sbin/"minidlnad", "-f", "minidlna.conf", "-p", port.to_s, "-P",
                              testpath/"minidlna.pid"
     sleep 2
 
     begin
-      assert_match /MiniDLNA #{version}/, shell_output("curl localhost:8081")
+      assert_match /MiniDLNA #{version}/, shell_output("curl localhost:#{port}")
     ensure
       Process.kill("SIGINT", File.read("minidlna.pid").to_i)
```

Comment:
Can you remove this as well, since brew kills any leftover test processes nowadays.

Reply:
Done